src/platform/user/authorization/components/IdentityNotVerified.jsx

```diff
@@ -4,8 +4,19 @@ import { AUTH_EVENTS } from '@department-of-veterans-affairs/platform-user/authe
 import PropTypes from 'prop-types';
 import recordEvent from '~/platform/monitoring/record-event';
 
+const HowToVerifyLink = () => (
+  <p className="vads-u-margin-y--4">
+    <va-link
+      href="/resources/verifying-your-identity-on-vagov/"
+      text="Learn how to verify your identity on VA.gov"
+      data-testid="verify-identity-link"
+    />
+  </p>
+);
+
 const IdentityNotVerified = ({
   headline = 'Verify your identity to view your complete profile',
+  postAlertChildren,
```

Comment:
I'd personally lean toward using the boolean prop and just removing `postAlertChildren` altogether since it's not actually used other than to pass a space to it.

maybe something like `showHowToVerifyLink` for a boolean prop maybe?

Reply:
`postAlertChildren` is there for a content tweak/override that is under [content review](https://github.com/department-of-veterans-affairs/va.gov-team/issues/73401). I guess since there are many possible outcomes, I could change to a boolean for now.

Reply:
Replace with a `showHelpContent` boolean for this PR